servers/src/common/types.rs

```diff
@@ -163,18 +163,21 @@ impl ServerConfig {
 		// check [server.p2p_config.capabilities] with 'archive_mode' in [server]
 		if let Some(archive) = self.archive_mode {
 			// note: slog not available before config loaded, only print here.
-			if archive != self
-				.p2p_config
-				.capabilities
-				.contains(p2p::Capabilities::FULL_HIST)
+			if archive
+				!= self
+					.p2p_config
+					.capabilities
+					.contains(p2p::Capabilities::FULL_HIST)
 			{
 				// if conflict, 'archive_mode' win
 				self.p2p_config
 					.capabilities
 					.toggle(p2p::Capabilities::FULL_HIST);
-				println!(
+				warn!(
```

Comment:
Hey 😺   Did you see the comment at line 178?  
```
"note: slog not available before config loaded, only print here."
```
I prefer to remove this warning message since it make user confusing.